addons/l10n_fr_account/data/template/account.account-fr.csv

```diff
@@ -512,6 +512,7 @@
 "pcg_6583","Losses resulting from indexation clauses","658300","expense","account.account_tag_operating","False","","Malis provenant de clauses d'indexation"
 "pcg_6584","Lots","658400","expense","account.account_tag_operating","False","","Lots"
 "pcg_6588","Setting up or winding up trusts","658800","expense","","False","","Opérations de constitution ou liquidation des fiducies"
+"pcg_658","Penalties and other expenses","658000","expense","","False","","Pénalités et autres charges"
```

Comment:
You have the group [here](https://github.com/odoo-dev/odoo/blob/4e821877c67f055f7897f0cf6b78a887057d94a3/addons/l10n_fr_account/data/template/account.group-fr.csv#L140)

Reply:
Wanted to stay consistent with https://github.com/odoo/odoo/commit/4bee26fca2e807dae59ee19a081b527473ce21f0
It's the exact same.
We could decide 658001 or 6589 if we prefer